docs/pages/contrib/pytest_plugins.rst

```diff
@@ -60,6 +60,32 @@ This is how it works internally:
   It does not affect production code.
 
 
+has_trace
+~~~~~~~~~
+
+Sometimes we have to know if a container is created correctly in a specific
+point of our flow.
+
+``has_trace`` helps us to check exactly this by identifying when a container is
+created and looking for the desired function.
+
+.. code:: python
+
+  >>> from returns.result import Result, Success, Failure
+
+  >>> def desired_function(arg: str) -> Result[int, str]:
+  ...     if arg.isnumeric():
+  ...         return Success(int(arg))
+  ...     return Failure('"{0}" is not a number'.format(arg))
+
+  >>> def test_if_failure_is_created_at_convert_function(returns):
```

Comment:
These test functions right now are not called. Am I right?
Or does `pytest` execute them?

Reply:
You're right, `pytest` does not execute them!

Reply:
Then, let's call them manually!

Reply:
How can I do this?? Because we're not able to "instantiate" the fixtures manually like:
```python
from returns.contrib.pytest.plugin import returns as returns_plugin, _patch_containers

returns_fixture = returns_plugin(_patch_containers())
```
Output:
```text
Fixture "_patch_containers" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
```

Reply:
See https://docs.pytest.org/en/stable/doctest.html#using-fixtures

Reply:
Done!